code/game/objects/items/defibrillator.dm

```diff
@@ -99,7 +99,7 @@
 		return
 	for(var/g in GLOB.observer_list)
 		var/mob/dead/observer/ghost = g
-		if(ghost.mind.current != src)
+		if(!ghost?.mind.current == src)
 			continue
```

Comment:
Why .? here

Reply:
Explain that this is in case of hard dels leaving nulls and remove said nulls from the list

Reply:
thats me being dumb, it was supposed to be on mind?.current